### ISSUE TITLE
Junitxml output returns multiple files 

### ISSUE THREAD

**Issue by secustor** [bridgecrewio/checkov#518]
**Describe the bug**
If multiple frameworks are triggered, checkov will print two separate junit.xml files to the output. This prevents Gitlab to parse the artifact as the XML is malformed. 

**To Reproduce**
1. Prepare a folder containing files of two frameworks
2. Run:
   a. `- checkov -d . --repo-id $CI_PROJECT_NAMESPACE/$CI_PROJECT_NAME --branch $CI_COMMIT_REF_NAME -s`
   b  `- checkov -d . -o junitxml > checkov.xml`
3. This results in following file: 

``` xml
<?xml version="1.0" ?>
<testsuites failures="3" errors="0" tests="5" disabled="0" time="0.0">
	<testsuite name="Ensure RBAC is enabled on AKS clusters" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSRbacEnabled">
		<testcase name="terraform Ensure RBAC is enabled on AKS clusters azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSRbacEnabled" file="/aks.tf"/>
	</testsuite>
	<testsuite name="Ensure Kube Dashboard is disabled" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSDashboardDisabled">
		<testcase name="terraform Ensure Kube Dashboard is disabled azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSDashboardDisabled" file="/aks.tf"/>
	</testsuite>
	<testsuite name="Ensure AKS has an API Server Authorized IP Ranges enabled" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSApiServerAuthorizedIpRanges">
		<testcase name="terraform Ensure AKS has an API Server Authorized IP Ranges enabled azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSApiServerAuthorizedIpRanges" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS has an API Server Authorized IP Ranges enabled&quot;"/>
		</testcase>
	</testsuite>
	<testsuite name="Ensure AKS cluster has Network Policy configured" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSNetworkPolicy">
		<testcase name="terraform Ensure AKS cluster has Network Policy configured azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSNetworkPolicy" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS cluster has Network Policy configured&quot;"/>
		</testcase>
	</testsuite>
	<testsuite name="Ensure AKS logging to Azure Monitoring is Configured" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSLoggingEnabled">
		<testcase name="terraform Ensure AKS logging to Azure Monitoring is Configured azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSLoggingEnabled" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS logging to Azure Monitoring is Configured&quot;"/>
		</testcase>
	</testsuite>
</testsuites>

<?xml version="1.0" ?>
<testsuites failures="0" errors="0" tests="1" disabled="0" time="0.0">
	<testsuite name="The default namespace should not be used" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.kubernetes.checks.DefaultNamespace">
		<testcase name="kubernetes The default namespace should not be used ConfigMap.nginx-ingress-grafana-dashboard.monitoring" classname="checkov.kubernetes.checks.DefaultNamespace" file="/config/monitoring/dashboards/nginxIngressDashboardConfigMap.yml"/>
	</testsuite>
</testsuites>
```

**Expected behavior**
The output should look like this: 

``` xml
<?xml version="1.0" ?>
<testsuites failures="3" errors="0" tests="6" disabled="0" time="0.0">
	<testsuite name="Ensure RBAC is enabled on AKS clusters" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSRbacEnabled">
		<testcase name="terraform Ensure RBAC is enabled on AKS clusters azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSRbacEnabled" file="/aks.tf"/>
	</testsuite>
	<testsuite name="Ensure Kube Dashboard is disabled" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSDashboardDisabled">
		<testcase name="terraform Ensure Kube Dashboard is disabled azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSDashboardDisabled" file="/aks.tf"/>
	</testsuite>
	<testsuite name="Ensure AKS has an API Server Authorized IP Ranges enabled" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSApiServerAuthorizedIpRanges">
		<testcase name="terraform Ensure AKS has an API Server Authorized IP Ranges enabled azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSApiServerAuthorizedIpRanges" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS has an API Server Authorized IP Ranges enabled&quot;"/>
		</testcase>
	</testsuite>
	<testsuite name="Ensure AKS cluster has Network Policy configured" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSNetworkPolicy">
		<testcase name="terraform Ensure AKS cluster has Network Policy configured azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSNetworkPolicy" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS cluster has Network Policy configured&quot;"/>
		</testcase>
	</testsuite>
	<testsuite name="Ensure AKS logging to Azure Monitoring is Configured" disabled="0" failures="1" errors="0" skipped="0" time="0" tests="1" package="checkov.terraform.checks.resource.azure.AKSLoggingEnabled">
		<testcase name="terraform Ensure AKS logging to Azure Monitoring is Configured azurerm_kubernetes_cluster.monitoring" classname="checkov.terraform.checks.resource.azure.AKSLoggingEnabled" file="/aks.tf">
			<failure type="failure" message="Resource &quot;azurerm_kubernetes_cluster.monitoring&quot; failed in check &quot;Ensure AKS logging to Azure Monitoring is Configured&quot;"/>
		</testcase>
	</testsuite>
	<testsuite name="The default namespace should not be used" disabled="0" failures="0" errors="0" skipped="0" time="0" tests="1" package="checkov.kubernetes.checks.DefaultNamespace">
		<testcase name="kubernetes The default namespace should not be used ConfigMap.nginx-ingress-grafana-dashboard.monitoring" classname="checkov.kubernetes.checks.DefaultNamespace" file="/config/monitoring/dashboards/nginxIngressDashboardConfigMap.yml"/>
	</testsuite>
</testsuites>
```

Environment: 
  - Gitlab Ci is setup following this guide: https://docs.bridgecrew.io/docs/integrate-with-gitlab-runner#in-gitlab
  - checkov Docker image
  - checkov version: `1.0.497`

**Comment by ampc** [comment]
I've also encountered this problem today.

**Comment by schosterbarak** [comment]
@ampc @secustor thank you for reporting this. this is now fixed. 
this was fixed at: https://github.com/bridgecrewio/checkov/pull/705
